data/json/items/book/cooking.json

```diff
@@ -3,13 +3,13 @@
     "id": "adv_chemistry",
     "type": "BOOK",
     "name": "Advanced Physical Chemistry",
-    "name_plural": "Advanced Physical Chemistry",
+    "name_plural": "Advanced Chemistry Textbooks",
```

Comment:
```suggestion
    "name_plural": "copies of Advanced Physical Chemistry",
```

Reply:
uh, I saw something in the discord about less characters being better... the alternative you've provided is perfectly nice by me. Thanks for the help :)